Whiskey/en-US/about_Whiskey_Writing_Tasks.help.txt

```diff
@@ -0,0 +1,359 @@
+TOPIC

```

Comment:
Remove this file. This documentation was moved to https://github.com/webmd-health-services/Whiskey/wiki/Writing-Tasks . If you made any changes to this file, make them to that page instead.